frontend/src/lib/components/PropertyFilters/PropertyFilters.tsx

```diff
@@ -59,6 +79,7 @@ export function PropertyFilters({
                             index={index}
                             totalCount={filtersWithNew.length - 1} // empty state
                             filters={filtersWithNew}
+                            pinnedFilters={pinnedFilters}
```

Comment:
Is this codepath actually doing anything?

Reply:
Another argument for splitting the PR :)